pxr/usd/lib/sdf/wrapFileFormat.cpp

```diff
@@ -129,6 +130,14 @@ void wrapFileFormat()
         .def("GetFileExtension", &This::GetFileExtension)
         .staticmethod("GetFileExtension")
 
+        .def("FindAllFileFormatExtensions", &This::FindAllFileFormatExtensions,
+             return_value_policy<TfPySequenceToTuple>())
```

Comment:
I think using TfPySequenceToList to return the results as a list instead of a tuple would be more conventional.